### ISSUE TITLE
Allow language description or metadata field

### ISSUE THREAD

**Issue by jdimeo** [tolgee/tolgee-platform#1791]
As a workaround for #1623 , we could do our own word counts for languages and update the language itself with a "description" field. We're considering hijacking Original name for this purpose, but it would be great to have an open-ended text field, ideally with basic HTML support for tables, simple formatting, etc. so we can describe the language according to our business needs:
![image](https://github.com/tolgee/tolgee-platform/assets/1071438/b9060235-0aac-4e99-89d7-2681d4ccc54b)

**Comment by JanCizmar** [comment]
Thanks! It sounds doable :)

**Comment by JanCizmar** [comment]
Hey John, is this still required?

**Comment by jdimeo** [comment]
Yes please. We still would like a way to track some custom metadata by language.

**Comment by JanCizmar** [comment]
OK, Adding to TODO.

**Comment by jdimeo** [comment]
Thanks @JanCizmar - I provided a screenshot in the original post how we are using a language's original name for this purpose

**Comment by JanCizmar** [comment]
Yean, but it's not clear to me how, would you like to show

> basic HTML support for tables, simple formatting

**Comment by jdimeo** [comment]
I guess it could just be Markdown like project description. I would use that to show many different "stats" about our languages since we have advanced use cases

**Comment by JanCizmar** [comment]
I have to pass this to @ZuzanaOdstrcilova for design proposal. I don't feel brave enough to implement this now.

**Comment by JanCizmar** [comment]
I have to pass this to @ZuzanaOdstrcilova for design proposal. I don't feel brave enough to implement this without proposal.
